core/block_dragger.js

```diff
@@ -186,6 +188,16 @@ Blockly.BlockDragger.prototype.startBlockDrag = function(currentDragDeltaXY,
   }
 };
 
+/**
+ * Fire a ui event at the start of a block drag.
```

Comment:
ui events are referenced as ``UI events`` elsewhere. Let's keep that consistent.